### DISCUSSION TITLE
How to get a specific ip address

### DISCUSSION THREAD

**Discussion by clumbo** [bpg/terraform-provider-proxmox#304] [Q&A] [answered]
Hi,

How do I get the ip address of eth0 from creating a vm to call ansible with the -i option as a local-exec?

Thanks

**Comment by bpg-dev** [comment, marked as answer]
Hey @clumbo! 👋🏼

If you take an example from https://github.com/bpg/terraform-provider-proxmox/issues/211#issuecomment-1509422435 and add something like  
```terraform
output "vm_ip" {
  value = proxmox_virtual_environment_vm.ubuntu_vm.ipv4_addresses[1]
}
```
Then it will print out IP(s) of `eth0`
```
Outputs:

vm_ip = tolist([
  "10.x.x.x",
])
```
(`ipv4_addresses[0]` is a `lo` interface)

It worth to mention that you'd need to have `qemu-guest-agent` installed on your VM and enabled in the template in order to let PVE read the network configuration of the VM. 

Alternatively you can use a static IP with the VM.

**Reply by clumbo** [reply]
Thanks for the examples
